### PR TITLE
Resource update

### PR DESCRIPTION
**Pull request by ShellyXueHan**
### changes include:
- [x] increased SSO pods resources
- [x] increased SSO DB statefulset resources
- [x] added a job template to initialize new SSO instance, specific for sandbox setup
- [x] ansible playbook to run custom changes in dev/test/prod
- [ ] oc template for running ansible playbook

@patricksimonian to update the above list with more details.

### CODE REVIEW

File: openshift/sso74-x509.yaml [hunk 1]
```diff
@@ -334,7 +335,19 @@ parameters:
...
-  value: 1Gi
+  value: 2Gi
+- description: Container memory request.
+  displayName: Container Memory Request
+  name: MEMORY_REQUEST
+  value: 2Gi
+- description: Container CPU limit.
+  displayName: Container CPU Limit
+  name: CPU_LIMIT
+  value: 2
```
Comment (patricksimonian) [inline openshift/sso74-x509.yaml:346]: think this needs to be a string

File: openshift/sso74-x509.yaml [hunk 2]
```diff
@@ -334,7 +335,19 @@ parameters:
...
+- description: Container CPU request.
+  displayName: Container CPU Request
+  name: CPU_REQUEST
+  value: 1
```
Comment (patricksimonian) [inline openshift/sso74-x509.yaml:350]: string value instead of int

File: openshift/sso74-x509.yaml (hunk 1 above)
Comment (patricksimonian) [inline openshift/sso74-x509.yaml:346]:
```suggestion
  value: "2"
```

File: openshift/sso74-x509.yaml (hunk 2 above)
Comment (patricksimonian) [inline openshift/sso74-x509.yaml:350]:
```suggestion
  value: "1"
```

**Comment by patricksimonian** [comment]
should i add my playbooks to this PR?

**Comment by ShellyXueHan** [comment]
> should i add my playbooks to this PR?

sure

**Comment by patricksimonian** [comment]
there are known issues with the playbook that need to be resolved

**Pull request merged**